dolphin-dnn/src/main/java/edu/snu/dolphin/dnn/conf/PoolingLayerConfigurationBuilder.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2016 Seoul National University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.snu.dolphin.dnn.conf;
+
+import edu.snu.dolphin.dnn.layerparam.initializer.LayerParameterInitializer;
+import edu.snu.dolphin.dnn.layerparam.initializer.PoolingLayerParameterInitializer;
+import edu.snu.dolphin.dnn.layers.PoolingLayer;
+import edu.snu.dolphin.dnn.layers.LayerBase;
+import edu.snu.dolphin.dnn.proto.NeuralNetworkProtos;
+import org.apache.reef.tang.Configuration;
+import org.apache.reef.tang.Tang;
+import org.apache.reef.util.Builder;
+
+/**
+ * Configuration builder for Pooling layer.
+ *
+ * The configuration that this builder generates is used to create a pooling layer instance.
+ * The generate configuration need to bind the parameter for a layer input shape, to inject layer instance.
+ */
+public final class PoolingLayerConfigurationBuilder implements Builder<Configuration> {
+
+  public static PoolingLayerConfigurationBuilder newConfigurationBuilder() {
+    return new PoolingLayerConfigurationBuilder();
+  }
+
+  private String poolingType = "MAX";
+  private int strideHeight = 1;
+  private int strideWidth = 1;
```

Comment:
The default values for `poolingType`, `strideHeight`, and `strideWidth` should be set in `neural_network.proto`. Otherwise, we should check whether these parameters are set or not, which makes the code complicated.